jsdoc3-maven-plugin/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>com.phasebash.jsdoc</groupId>
         <artifactId>jsdoc3</artifactId>
-        <version>1.1.0</version>
+        <version>1.1.1-SNAPSHOT</version>
     </parent>
```

Comment:
This is a big upgrade, so the semantic version should communicate this.  Let's use 1.2.0 instead.